cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/VerifyBundle.kt

```diff
@@ -107,6 +108,16 @@ open class VerifyBundle @Inject constructor(objects: ObjectFactory) : DefaultTas
         }
     }
 
+    private fun verifyReservedPackage(verifier: Verifier) {
+        val reservedPackageName = Regex("^net.corda(\\..+)?$")
+        verifier.exportPackage
+            .keyList()
+            .asSequence()
+            .plus(verifier.privatePackage.keyList())
```

Comment:
Private packages are not the same as exported packages, which makes these distinct violations. These two should therefore generate different error messages. I was expecting something more like:
```
verifyImportPackage(verifier)
verifyExportPackage(verifier)
verifyPrivatePackage(verifier)
```
Obviously the "package name matching" logic can still be shared.

Reply:
I've split the method into two and shared the package matching logic